en/deploy/README.md

```diff
@@ -88,7 +88,7 @@ $ git commit -m "My Django Girls app, first commit"
 
 ## Pushing your code to GitHub
 
-Go to [GitHub.com](https://www.github.com) and sign up for a new, free user account. (If you already did that in the workshop prep, that is great!) Be sure to remember your password (add it to your password manager, if you use one).
+Go to [GitHub.com](https://www.github.com) and sign up for a new, free user account. (If you already did that in the workshop prep, that is great!) Be sure to remember your password. (Add it to your password manager, if you use one.)
```

Comment:
Should we drop the parenthesis?

Reply:
How about for the second sentence:

Be sure to remember your password -- add it to your password manager, if you use one.